units/patch_intent_test.go

```diff
@@ -656,6 +659,79 @@ func (s *PatchIntentUnitsSuite) TestProcessCliPatchIntent() {
 	s.Require().Empty(out)
 }
 
+func (s *PatchIntentUnitsSuite) TestUpdatePatchDocPatches() {
+	githubOauthToken, err := s.env.Settings().GetGithubOauthToken()
+	s.Require().NoError(err)
+
+	flags := evergreen.ServiceFlags{
+		GithubPRTestingDisabled: true,
+	}
+	s.NoError(evergreen.SetServiceFlags(flags))
+
+	patchContent, _, err := thirdparty.GetGithubPullRequestDiff(context.Background(), githubOauthToken, s.githubPatchData)
+	s.Require().NoError(err)
+
+	patchedParserProject := `
+modules:
+  - name: sandbox
```

Comment:
Can we add another module so we hit both the "exists in the patch already" and the "doesn't exist in the patch" case?